physcraper/__init__.py

```diff
@@ -3162,7 +2952,7 @@ def sp_dict(self, downtorank=None):
                         downtorank_name = self.ids.ncbi_parser.get_name_from_id(downtorank_id)
                     tax_name = downtorank_name
                     tax_id = downtorank_id
-                tax_name = tax_name.replace(" ", "_")
+                tax_name = tax_name.replace(" ", "_") #TODO we should store the names with underscores in the dicts
```

Comment:
That is why we replace the " " with "_" before we store it in the dict. Names with " " come from ncbi

Reply:
Yeah - I think in this case the spaces should already be replaced with underscores when we get the tax_name here, because we already fixed them in the dict and the search classes? But it shouldn't do any harm to double check.